src/applications/mhv-landing-page/components/alerts/AlertMhvUserAction.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+// eslint-disable-next-line import/no-named-default
+import { default as recordEventFn } from '~/platform/monitoring/record-event';
+
+const AlertMhvUserAction = ({ errorCode, testId, recordEvent }) => {
```

Comment:
Notice that the alert icon is getting squashed on smaller width:
<img width="592" alt="Screenshot 2024-12-18 at 11 10 07 AM" src="https://github.com/user-attachments/assets/f4dc34a2-0507-46bc-afca-e219abab23b9" />